scripts/opa/main.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/open-policy-agent/opa/ast"
+	"github.com/open-policy-agent/opa/cmd"
+	"github.com/open-policy-agent/opa/rego"
+	"github.com/open-policy-agent/opa/types"
+
+	parsers "github.com/snyk/snyk-iac-parsers"
+	terraform "github.com/snyk/snyk-iac-parsers/terraform"
+)
+
+func runOpa() {
+	rego.RegisterBuiltin1(
+		&rego.Function{
+			Name:    "hcl2.unmarshal_file",
+			Decl:    types.NewFunction(types.Args(types.S), types.A),
+			Memoize: true,
+		},
+		func(bctx rego.BuiltinContext, a *ast.Term) (*ast.Term, error) {
+
+			var filePath string
+
+			if err := ast.As(a.Value, &filePath); err != nil {
+				return nil, err
+			}
+
+			content, err := ioutil.ReadFile(filePath)
+			if err != nil {
+				return nil, err
+			}
+
+			input, err := terraform.ParseHclToJson("test.tf", string(content), terraform.VariableMap{})
+			if err != nil {
+				return nil, err
+			}
+
+			var parsedInput terraform.JSON
```

Comment:
nit: Similarly to this comment, From what I can tell it seems `input` already contains the parsed content in raw JSON format, and that `parsedInput` contains the processed, unmarshalled content.

Do you think it'd be clearer to change `input` to `rawParsedInput`, or `jsonParsedInput`?

Sharing my thoughts about it, and the choice of whether to change or not is entirely up to you.